broker-daemon/block-order-worker/index.js

```diff
@@ -402,55 +416,69 @@ class BlockOrderWorker extends EventEmitter {
       throw new Error(`No engine available for ${counterSymbol}`)
     }
 
-    const promisedFills = orders.map(async (order) => {
+    const ordersByOrderId = await Promise.all(orders.map((order) => {
+      const range = {
+        gte: order.orderId,
+        lte: order.orderId
+      }
+      return getRecords(this.ordersByOrderId, Order.fromStorage.bind(Order), this.ordersByOrderId.range(range))
+    }))
+
+    const ownOrderIds = ordersByOrderId.filter(order => order && order.length > 0).map(([order]) => order.orderId)
```

Comment:
confusing to use `order` here for an array of orders, better to use something like `orders`